synapseAnnotations/data/analysis.json

```diff
@@ -1,388 +1,340 @@
+        "description": "Conditional independence graph construction from gene expression using an L2^2 penalized regression neighborhood selection method.",
+        "source": "https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-10-384"
+      },
       {
-        "value": "genie3", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "genie3",
+        "description": "Gene expression network reconstruction using random forests.",
+        "source": "http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0012776"
+      },
       {
-        "value": "tigress", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "tigress",
+        "description": "Gene expression network reconstruction using stability selection.",
+        "source": "https://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-6-145"
+      },
       {
-        "value": "aracne", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "aracne",
+        "description": "Gene expression network reconstruction using mutual information.",
+        "source": "https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-7-S1-S7"
+      },
       {
-        "value": "mrnet", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "mrnet",
+        "description": "An information theoretic method to infer gene expression networks.",
+        "source": "https://www.ncbi.nlm.nih.gov/pubmed/18354736"
+      },
       {
-        "value": "c3net", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "c3net",
+        "description": "Method to infer causal relationships in gene expression networks with mutual information.",
+        "source": "https://bmcsystbiol.biomedcentral.com/articles/10.1186/1752-0509-4-132"
+      },
       {
-        "value": "MEGENA", 
-        "description": "", 
-        "source": ""
+        "value": "MEGENA",
+        "description": "Multiscale Embedded Gene Co-expression Network Analysis",
+        "source": "http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004574"
       },
       {
-        "value": "WINA", 
-        "description": "", 
+        "value": "WINA",
+        "description": "",
         "source": ""
       }
     ]
-  }, 
+  },
   {
-    "name": "differentialExpressionMethod", 
-    "description": "", 
-    "columnType": "STRING", 
-    "maximumSize": 250, 
+    "name": "differentialExpressionMethod",
+    "description": "Method used to test for differentially expressed genes or isoforms.",
+    "columnType": "STRING",
+    "maximumSize": 250,
     "enumValues": [
       {
-        "value": "limma", 
-        "description": "", 
-        "source": ""
-      }, 
+        "value": "limma",
+        "description": "Data analysis, linear models and differential expression for microarray data.",
+        "source": "http://www.ebi.ac.uk/efo/swo/SWO_0000593"
```

Comment:
Source 404s

Reply:
This is an issue with SWO, @kdaily created an issue on their repo. For now can you put the full query link?